From 6cefc7df1a6158c79d23029df183c09b10b88cad Mon Sep 17 00:00:00 2001
From: David Robillard <d@drobilla.net>
Date: Sat, 13 Mar 2021 14:12:48 -0500
Subject: [PATCH] Simplify rdfs:label restrictions

Using someValuesFrom here is redundant and overly complex, since the property
range specifies that it is for literals anyway.  So, simply replace it with a
minCardinality of 1.
---
 lv2/presets/presets.ttl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lv2/presets/presets.ttl b/lv2/presets/presets.ttl
index 156f0813..60189ead 100644
--- a/lv2/presets/presets.ttl
+++ b/lv2/presets/presets.ttl
@@ -18,7 +18,7 @@ pset:Bank
 	rdfs:subClassOf [
 		a owl:Restriction ;
 		owl:onProperty rdfs:label ;
-		owl:someValuesFrom xsd:string ;
+		owl:minCardinality 1 ;
 		rdfs:comment "A Bank MUST have at least one string rdfs:label."
 	] ;
 	rdfs:comment "A bank of presets." .
@@ -31,7 +31,7 @@ pset:Preset
 	rdfs:subClassOf [
 		a owl:Restriction ;
 		owl:onProperty rdfs:label ;
-		owl:someValuesFrom xsd:string ;
+		owl:minCardinality 1 ;
 		rdfs:comment "A Preset MUST have at least one string rdfs:label."
 	] .
 
